pandas/core/reshape/tile.py

```diff
@@ -327,6 +331,13 @@ def qcut(
     bins : ndarray of floats
         Returned only if `retbins` is True.
 
+    See Also
+    --------
+    cut : Transform values into pre-defined or equal-width intervals (bins).
+    Categorical : Array type for storing data that come from a
+        fixed set of values.
+    Series : One-dimensional array with axis labels (including time series).
```

Comment:
not sure how a reference to Series is relevant here (its true this can return a series but this is cluttering things)

Reply:
Done, updated! Series - dropped, Wiki reference - added.